src/functTest/groovy/com/bmuschko/gradle/docker/DockerRemoteApiPluginFunctionalTest.groovy

```diff
@@ -261,6 +261,38 @@ class DockerRemoteApiPluginFunctionalTest extends AbstractGroovyDslFunctionalTes
         build('convert')
     }
 
+
+    def "configuration cache compatible when docker state changes on disk for OS #osName"() {
```

Comment:
This test method should define a precondition for the operating system.

Reply:
Can you please elaborate? I am not sure i follow. 
The reason that I added the OS was the knowledge of the implementation and that the path to be taken is only achieved by specifying OS.

Reply:
This: `@IgnoreIf({ os.windows })`. If other developers run the test on a Windows machine, it will likely fail.

Reply:
Ah, sure!

Reply:
done